tests/dragonfly/pymemcached_test.py

```diff
@@ -67,10 +68,17 @@ def test_noreply_pipeline(df_server: DflyInstance, memcached_client: MCClient):
     # check all commands were executed
     assert memcached_client.get_many(keys) == {k: v.encode() for k, v in zip(keys, values)}
 
-    info = Redis(port=df_server.port).info()
+    client = df_server.client()
+    info = await client.info()
     if info["total_pipelined_commands"] == 0:
-        warnings.warn("No pipelined commands were detected. Info: \n" + str(info))
-        assert False, "No pipelined commands were detected."
+        logging.error("No pipelined commands were detected. Info: \n" + str(info))
+
+        # Try again
+        for k, v in zip(keys, values):
```

Comment:
what;'s the reason to try again and then fail the assertion?

Reply:
because I want to manually inspect the results

Reply:
when it happens